default.env

```diff
@@ -1,6 +1,10 @@
-LOG_LEVEL=trace
-USERID_HEADER_KEY=userid
-GROUPS_HEADER_KEY=usergroups
-CLIENTTYPE_HEADER_KEY=clienttype
-BACKOFFICE_HEADER_KEY=isbackoffice
-MICROSERVICE_GATEWAY_SERVICE_NAME=microservice-gateway
```

Comment:
These variables are required from the custom-plugin-lib. I think they should be kept.

Reply:
added it back